course/README.md

```diff
@@ -4,4 +4,6 @@
 
 ## [Kanban board](https://trello.com/b/xb15aXBr/kanban)
 
-## [Whiteboard](https://docs.google.com/document/d/1sXhC2zKXtrObzM79ByO6NsMDnWDgGQCGQxRmBBcsqYM/edit)
\ No newline at end of file
+## [Whiteboard](https://docs.google.com/document/d/1sXhC2zKXtrObzM79ByO6NsMDnWDgGQCGQxRmBBcsqYM/edit)
+
+### Adding changes on my own
```

Comment:
REMOVE this line. may it work to replace the contents of this file with the original one (fetch it from upstream/develop) and copy-paste the contents here to reset it.